### PR TITLE
Nestornav/grunt init task

### PR DESCRIPTION
**Pull request by nestornav**
Set a new grunt task called "init" for the start up up the project. This task install the bower components for editor and run the test suit.

### CODE REVIEW

File: gruntfile.js [hunk 1]
```diff
@@ -94,12 +94,13 @@ module.exports = function (grunt) {
...
     grunt.registerTask('travis', ['jshint', 'test']);
 
-    grunt.registerTask('init','Prepare a fresh install to start working with Puma',function(){
+    grunt.registerTask('init','Prepare to start working with Puma',function(){
         var exec = require('child_process').exec;
         var done = this.async();
         exec('bower install', {cwd: './editor'}, function(err, stdout, stderr) {
             console.log(stdout);
             done();
+            grunt.task.run('test');
```
Comment (emravera) [inline gruntfile.js]: We are performing init only. I think this is not necessary.

File: gruntfile.js [hunk 2]
```diff
@@ -83,6 +82,9 @@ module.exports = function (grunt) {
         }
     });
 
+    //Load the plugins for project start up
+    //grunt.loadNpmTasks('grunt-npm-install');
```
Comment (emravera) [inline gruntfile.js]: Remove commented code

File: package.json [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
-    },
-    "license": "UTN-LIS",
-    "devDependencies": {
-        "grunt-contrib-clean": "~0.7.0",
-        "grunt-contrib-jshint": "~0.11.3",
-        "grunt-contrib-qunit": "~0.5.1",
-        "grunt-contrib-requirejs": "0.4.4",
-        "grunt": "~0.4.5",
-        "jshint-stylish": "~1.0.1",
-        "load-grunt-tasks": "~3.1.0"
```
Comment (emravera) [inline package.json]: This dev dependency is esential for the grunt task to run. It is missing from the package.json and for that reason all the travis CI jobs are failing.

**Comment by emravera** [comment]
Please add the the missing dependency so we can check if the build start by executing grunt travis.

**Comment by nestornav** [comment]
I added the missing dependency.

**Pull request merged**